src/components/Cluster/NewCluster/CreateRegularCluster.js

```diff
@@ -68,14 +67,8 @@ const FlexWrapperAZDiv = styled.div`
 `;
 
 class CreateRegularCluster extends React.Component {
-  static isScalingAutomatic(provider, releaseVer) {
-    if (provider !== Providers.AWS) {
-      return false;
-    }
-
-    // In order to have support for automatic scaling and therefore for scaling
-    // limits, provider must be AWS and cluster release >= 6.3.0.
-    return cmp(releaseVer, '6.2.99') === 1;
+  static isScalingAutomatic(provider) {
+    return provider !== Providers.AWS;
```

Comment:
This should be the other way around. Autoscaling is only available on AWS at the moment, and this check is disabling it for AWS clusters

Reply:
good catch! :+1: thanks